### ISSUE TITLE
[Bug]: credits didn't recognized 

### ISSUE THREAD

**Issue by smartisanyyh** [jumoog/intro-skipper#249]
### Self service debugging

- [X] Jellyfin is updated (10.9.9) and my permissions are correct (or I did not use Docker)

### Describe the bug

There is no problem in skipping the intro, but the credits is not correctly identified.It seems that there is no comparison at all at the end of the movie, because there are only the first ten minutes of images in fingerprint management, blue and red.

also asked on [discord](https://discord.com/channels/1238234010568753172/1240674195101454526/1274733795601416242)

### Jellyfin install method

DSM 7.2.1-69057 Update 5 

### Container image/tag or Jellyfin version

10.9.9

### Operating System

DSM 7.2.1-69057 Update 5 

### Support Bundle

```shell
* Jellyfin version: 10.9.9
* Plugin version: 0.2.0+92aa5fef8806
* Queue contents: 346 episodes, 13 seasons
* Warnings: `None`
* FFmpeg: `okay`

FFmpeg version:

ffmpeg version 6.0.1-Jellyfin Copyright (c) 2000-2023 the FFmpeg developers
built with gcc 11 (Ubuntu 11.4.0-1ubuntu1~22.04)
configuration: --prefix=/var/packages/Jellyfin/target --target-os=linux --extra-version=Jellyfin --disable-doc --disable-ffplay --disable-ptx-compression --disable-static --disable-libxcb --disable-sdl2 --disable-xlib --enable-lto --enable-gpl --enable-version3 --enable-shared --enable-gmp --enable-gnutls --enable-chromaprint --enable-opencl --enable-libdrm --enable-libxml2 --enable-libass --enable-libfreetype --enable-libfribidi --enable-libfontconfig --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libopenmpt --enable-libdav1d --enable-libsvtav1 --enable-libwebp --enable-libvpx --enable-libx264 --enable-libx265 --enable-libzvbi --enable-libzimg --enable-libfdk-aac --arch=amd64 --enable-libshaderc --enable-libplacebo --enable-vulkan --enable-vaapi --enable-amf --enable-libvpl --enable-ffnvcodec --enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvdec --enable-nvenc
libavutil      58.  2.100 / 58.  2.100
libavcodec     60.  3.100 / 60.  3.100
libavformat    60.  3.100 / 60.  3.100
libavdevice    60.  1.100 / 60.  1.100
libavfilter     9.  3.100 /  9.  3.100
libswscale      7.  1.100 /  7.  1.100
libswresample   4. 10.100 /  4. 10.100
libpostproc    57.  1.100 / 57.  1.100
```
```


### Jellyfin logs

```shell
[2024-08-19 09:55:09.239 +08:00] [INF] [44] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: Scheduled Task is starting
[2024-08-19 09:55:09.480 +08:00] [INF] [44] ConfusedPolarBear.Plugin.IntroSkipper.QueueManager: Running enqueue of items in library "电影"
[2024-08-19 09:55:09.496 +08:00] [INF] [44] ConfusedPolarBear.Plugin.IntroSkipper.QueueManager: Running enqueue of items in library "节目"
[2024-08-19 09:55:09.625 +08:00] [INF] [44] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 36 files from "唐朝诡事录" season -1
[2024-08-19 09:55:09.626 +08:00] [INF] [42] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 40 files from "唐朝诡事录之西行" season -1
[2024-08-19 09:55:14.792 +08:00] [INF] [53] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.10.191" closed
[2024-08-19 09:55:15.459 +08:00] [INF] [45] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.10.191" request
[2024-08-19 09:55:16.516 +08:00] [INF] [19] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.10.191" closed
[2024-08-19 10:00:15.587 +08:00] [INF] [44] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 38 files from "玫瑰的故事" season -1
[2024-08-19 10:06:49.366 +08:00] [INF] [44] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 24 files from "老友记" season -1
[2024-08-19 10:07:56.740 +08:00] [INF] [42] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 23 files from "老友记" season -1
[2024-08-19 10:11:02.315 +08:00] [INF] [44] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 25 files from "老友记" season -1
[2024-08-19 10:12:32.719 +08:00] [INF] [42] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 24 files from "老友记" season -1
[2024-08-19 10:16:03.092 +08:00] [INF] [44] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 24 files from "老友记" season -1
[2024-08-19 10:17:20.947 +08:00] [INF] [60] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 25 files from "老友记" season -1
[2024-08-19 10:17:55.348 +08:00] [INF] [19] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.10.191" request
[2024-08-19 10:18:01.378 +08:00] [INF] [19] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.10.191" closed
[2024-08-19 10:20:42.431 +08:00] [INF] [44] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 24 files from "老友记" season -1
[2024-08-19 10:22:08.180 +08:00] [INF] [60] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 23 files from "老友记" season -1
[2024-08-19 10:25:33.818 +08:00] [INF] [44] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 23 files from "老友记" season -1
[2024-08-19 10:26:32.778 +08:00] [INF] [60] ConfusedPolarBear.Plugin.IntroSkipper.DetectIntrosCreditsTask: [Mode: Introduction] Analyzing 17 files from "老友记" season -1
[2024-08-19 10:30:37.835 +08:00] [INF] [44] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Detect Intros and Credits" Completed after 35 minute(s) and 28 seconds
[2024-08-19 10:56:03.331 +08:00] [INF] [55] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/volume2/Download/电影"
[2024-08-19 10:56:03.331 +08:00] [INF] [55] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/volume2/Download/电视剧"
[2024-08-19 10:56:03.339 +08:00] [INF] [55] Emby.Server.Implementations.Library.LibraryManager: Validating media library
[2024-08-19 10:56:03.657 +08:00] [WRN] [55] MediaBrowser.Controller.Entities.BaseItem: Library folder "/var/packages/Jellyfin/target/data/data/playlists" is inaccessible or empty, skipping
[2024-08-19 10:56:03.758 +08:00] [WRN] [36] MediaBrowser.Controller.Entities.BaseItem: Library folder "/var/packages/Jellyfin/target/data/data/playlists" is inaccessible or empty, skipping
[2024-08-19 10:56:31.903 +08:00] [INF] [21] Emby.Server.Implementations.ScheduledTasks.TaskManager: "扫描媒体库" Completed after 0 minute(s) and 28 seconds
[2024-08-19 10:56:32.134 +08:00] [INF] [45] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/volume2/Download/电影"
[2024-08-19 10:56:32.176 +08:00] [INF] [42] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/volume2/Download/电视剧"
[2024-08-19 10:56:51.904 +08:00] [INF] [45] ConfusedPolarBear.Plugin.IntroSkipper.Entrypoint: Initiate automatic analysis task.
[2024-08-19 10:56:52.172 +08:00] [INF] [45] ConfusedPolarBear.Plugin.IntroSkipper.QueueManager: Running enqueue of items in library "电影"
[2024-08-19 10:56:52.180 +08:00] [INF] [45] ConfusedPolarBear.Plugin.IntroSkipper.QueueManager: Running enqueue of items in library "节目"
[2024-08-19 12:54:10.957 +08:00] [INF] [17] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.10.191" request
[2024-08-19 12:54:24.841 +08:00] [INF] [22] ConfusedPolarBear.Plugin.IntroSkipper.Plugin: Unstable plugin version built from commit "92aa5fef88060ca953c10680d9603f2f78af0a0f"
```

**Comment by smartisanyyh** [comment]
fixed by Updated to 0.2.0.16

**Comment by smartisanyyh** [comment]
fixed by Updated to 0.2.0.16
